src/index.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  Network,
+  Environment,
+  OverrideType,
+  RTDBMetadata,
+  TokenInfo,
+  TokenInfoDTO,
+} from './types'
+import CeloMainnetTokensInfo from './data/mainnet/celo-tokens-info.json'
+import CeloTestnetTokensInfo from './data/testnet/celo-alfajores-tokens-info.json'
+import EthereumMainnetTokensInfo from './data/mainnet/ethereum-tokens-info.json'
+import EthereumTestnetTokensInfo from './data/testnet/ethereum-sepolia-tokens-info.json'
+import MainnetAddressesExtraInfo from './data/mainnet/addresses-extra-info.json'
+import TestnetAddressesExtraInfo from './data/testnet/addresses-extra-info.json'
+import AddressesExtraInfoSchema from './schemas/addresses-extra-info'
+import {
+  getNetworkToTokensInfoDTO,
+  transformCeloTokensForRTDB,
+} from './utils/transforms'
+import { RTDBAddressToTokenInfoSchema } from './schemas/tokens-info'
+
+export function getCeloRTDBMetadata(environment: Environment): RTDBMetadata[] {
+  const [tokensInfo, addressesExtraInfo] =
+    environment === 'mainnet'
+      ? [CeloMainnetTokensInfo, MainnetAddressesExtraInfo]
+      : [CeloTestnetTokensInfo, TestnetAddressesExtraInfo]
+  return [
+    {
+      data: addressesExtraInfo,
+      schema: AddressesExtraInfoSchema,
+      rtdbLocation: 'addressesExtraInfo',
+      overrideType: OverrideType.OverrideAll,
+    },
+    {
+      data: transformCeloTokensForRTDB(tokensInfo),
+      schema: RTDBAddressToTokenInfoSchema,
+      rtdbLocation: 'tokensInfo',
+      overrideType: OverrideType.DeleteMissingKeysAndUpdate,
+    },
+  ]
+}
+
+export function getTokensInfo(
+  environment: Environment,
+): Record<Network, TokenInfo[]> {
+  // just exported for testing
+  return environment === 'mainnet'
+    ? {
+        [Network.ethereum]: EthereumMainnetTokensInfo,
+        [Network.celo]: CeloMainnetTokensInfo,
+      }
+    : {
+        [Network.ethereum]: EthereumTestnetTokensInfo,
+        [Network.celo]: CeloTestnetTokensInfo,
+      }
+}
```

Comment:
Side note: I'd prefer for us to avoid mainnet / testnet grouping in code as much as possible. Unless there's no way around it. I made a small comment after the fact in https://github.com/valora-inc/address-metadata/pull/236#issuecomment-1702534629

Probably we should sync on that during and eng team meeting.

Reply:
added it to the agenda